lib/pure/base64.nim

```diff
@@ -16,44 +16,45 @@
 ## Each Base64 digit represents exactly 6 bits of data. Three 8-bit
 ## bytes (i.e., a total of 24 bits) can therefore be represented by
 ## four 6-bit Base64 digits.
-##
-## Basic usage
-## ===========
-##
+
+##[
+# Basic usage
+]##
+
+##[
```

Comment:
```suggestion

```
(This is an empty line, but GitHub doesn't render it for some reason.) Also, if you're already putting titles in multiline comments, why not the rest as well?